examples/counter_in/cont_acq_freq_wfm_impl_time.py

```diff
@@ -0,0 +1,37 @@
+"""Example of counter input frequency acquisition.
+
+This example demonstrates how to continuously measure frequency using one
+counter on a Counter Input Channel. The Edge, Minimum Value and
+Maximum Value are all configurable.
+"""
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, Edge, CounterFrequencyMethod, FrequencyUnits
+
+
+with nidaqmx.Task() as task:
+    chan = task.ci_channels.add_ci_freq_chan(
+        "Dev1/ctr0",
+        min_val=100.0,
+        max_val=1000.0,
+        units=FrequencyUnits.HZ,
+        edge=Edge.RISING,
+        meas_method=CounterFrequencyMethod.LOW_FREQUENCY_1_COUNTER,
```

Comment:
I would just remove this setting, its pretty advanced and largely irrelevant starting with X Series

```suggestion
```